classes/OccurrenceIndividual.php

```diff
@@ -162,6 +163,21 @@ public function setOccurData(){
 					if($this->occArr['collectioncode']){
 						if($this->metadataArr['collectioncode'] != $this->occArr['collectioncode']) $this->metadataArr['collectioncode'] = $this->occArr['collectioncode'];
 					}
+					// if($this->metadataArr['guidtarget'] == 'occurrenceID'){
```

Comment:
The old code had:
`elseif($this->metadataArr['guidtarget'] == 'symbiotaUUID'){
							if(isset($this->occArr['recordid'])) $this->occArr['occurrenceid'] = $this->occArr['recordid'];
						}`
						
The new spec calls for:
> if GUID source = symbiotaUUID, occurrenceID = IF omoccurrences.occurrenceID IS NOT NULL, omoccurrences.occurrenceID. Otherwise, occurrenceID = omoccurrences.recordID

I think that the old code would use recordId if it exists, whereas the new spec wants us to use occurrenceId preferentially if both happen to exist. Right?

Reply:
Got it. Thanks!

Currently, the occurenceID label doesn't display if there is nothing to display. Do we still want that behavior? Or do we want the label to appear now no matter what?

Reply:
@Atticus29 , I think it's fine for it to not display.

Reply:
Fixed in ac5ee9773.